core/app/services/spree/export.rb

```diff
@@ -0,0 +1,103 @@
+require 'nokogiri'
+
+module Spree
+  class Export
+    prepend Spree::ServiceModule::Base
+
+    def export_google_rss(options)
```

Comment:
I would suggest to rename this method name to `call`, it's ServiceObject so we should stick to convention